networkdb/networkdb.go

```
@@ -270,10 +286,25 @@ func (nDB *NetworkDB) CreateEntry(tname, nid, key string, value []byte) error {
 		return fmt.Errorf("cannot create entry in table %s with network id %s and key %s, already exists", tname, nid, key)
 	}
 
+	var epochID uint64
+	nDB.RLock()
+	thisNodeNetworks, ok := nDB.networks[nDB.config.NodeName]
+	if ok {
+		// The network may have been removed
+		network, networkOk := thisNodeNetworks[nid]
+		if !networkOk {
+			nDB.RUnlock()
+			return nil
```

should we return an error here?